src/use_case/login/LoginUserDataAccessInterface.java

```diff
@@ -0,0 +1,11 @@
+package use_case.login;
+
+import entity.User;
+
+public interface LoginUserDataAccessInterface {
+    boolean existsByName(String identifier);
+
+    void save(User user);
+
+    User get(String username);
```

Comment:
Change function signature to `User getUser(String username)`

Reply:
Will do the rename in the next PR.

Reply:
Renamed.